src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1386,4 +1405,31 @@ public static void onAdvancement(EntityPlayerMP player, Advancement advancement)
     {
         MinecraftForge.EVENT_BUS.post(new AdvancementEvent(player, advancement));
     }
+    
+    public static void loadFunctions(Map<ResourceLocation, FunctionObject> functions)
+    {
+        Map<ResourceLocation, FunctionObject> extraFunctions = new HashMap<>();
```

Comment:
Same as above, its already a data format, we should load the data not fire an event.

Reply:
Same as above, I don't want to load functions from code necessarily.